pkgs/applications/science/math/sage/threejs-sage.nix

```
@@ -12,7 +12,8 @@ stdenv.mkDerivation rec {
   };
 
   installPhase = ''
-    mkdir -p $out/lib/node_modules/three
-    cp -r build version $out/lib/node_modules/three
+    mkdir -p $out/lib/node_modules/three/$(cat version)
+    cp version $out/lib/node_modules/three
+    cp -r build/* $out/lib/node_modules/three/$(cat version)
```

```suggestion
    mkdir -p "$out/lib/node_modules/three/$(cat version)"
    cp version "$out/lib/node_modules/three"
    cp -r build/* "$out/lib/node_modules/three/$(cat version)"
```

Just as a best practice when command substitution is involved. If you agree, you can either apply the change through github (I'll then squash & merge) or apply it locally. Whatever is easier for you.

---

Agreed, thanks for the feedback! I've applied your fix and squashed the commits.